### PR TITLE
Save and load game fixes (and some other small fixes)

### PR DESCRIPTION
**Pull request by StraToN**
Fixes https://github.com/godot-escoria/escoria-issues/issues/73

### CODE REVIEW

File: addons/escoria-core/game/core-scripts/esc/types/esc_command.gd [hunk 1]
```diff
@@ -82,26 +82,32 @@ func _init(command_string):
...
-		var command_path = "%s/%s.gd" % [
-			base_path.trim_suffix("/"),
-			self.name
-		]
-		if ResourceLoader.exists(command_path):
-			command_found = true
-			
-	if not command_found:
+	if not command_exists():
+		return false
```
Comment (dploeger) [inline addons/escoria-core/game/core-scripts/esc/types/esc_command.gd:86]: This should probably go after the log output in the next few lines

File: addons/escoria-core/game/core-scripts/esc_room.gd [hunk 2]
```diff
@@ -110,13 +110,6 @@ func _ready():
 				player.update_idle()
 		escoria.object_manager.get_object("_camera").node.set_target(player)
 	
-	for n in get_children():
```
Comment (dploeger) [inline addons/escoria-core/game/core-scripts/esc_room.gd:113]: Why did you remove this? How are the room object's registered then?

**Review by dploeger — CHANGES_REQUESTED** [review]

**Comment by dploeger** [comment]
Also, the load selection still stays open after loading the screen:

image

**Comment by dploeger** [comment]
Also the loading from room 01 still doesn't start until the initial event finishes.

**Comment by dploeger** [comment]
Also, I noticed that when you load from within a game and then hit "menu" again, the loading screen is displayed directly instead of the pause menu.

File: addons/escoria-core/game/core-scripts/esc/types/esc_command.gd (hunk 1 above)
Comment (StraToN) [reply addons/escoria-core/game/core-scripts/esc/types/esc_command.gd:86]: Oops, indeed. Fixed

File: addons/escoria-core/game/core-scripts/esc_room.gd (hunk 2 above)
Comment (StraToN) [reply addons/escoria-core/game/core-scripts/esc_room.gd:113]: This block only registers children node that are `ESCLocation`s and `is_start_location`.
All `ESCItem`s register by themselves in their own `_ready()` function.

Comment (dploeger) [reply addons/escoria-core/game/core-scripts/esc_room.gd:113]: Ah right.

**Comment by StraToN** [comment]
@dploeger All issues you mentioned should now be resolved

**Review by dploeger — APPROVED** [review]
Now everything's stable. Great work! Thanks a lot 🎁

**Comment by StraToN** [comment]
🎉
